<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>860</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1192</width>
    <height>860</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1192</width>
    <height>860</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ndnSIMDataProcess_20171011</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Canvas" name="Canvas_1" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="dirButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>目录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dirField">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>881</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>./</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>291</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>数据文件列表</string>
    </property>
    <widget class="QListWidget" name="filesListBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>291</width>
       <height>351</height>
      </rect>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>131</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>字段列表</string>
    </property>
    <widget class="QListWidget" name="fieldsListBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>131</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Type列表</string>
    </property>
    <widget class="QListWidget" name="typesListBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>131</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="faceCheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>过滤</string>
    </property>
   </widget>
   <widget class="QComboBox" name="faceDropDown">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>271</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Face:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoNodeCheckBox">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>60</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>自动图形节点号</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DropDown_1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>节点</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>100</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>节点</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DropDown_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>100</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="Canvas" name="Canvas_2" native="true">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>130</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>节点</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DropDown_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>410</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="Canvas" name="Canvas_3" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="DropDown_4">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>410</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>410</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>节点</string>
    </property>
   </widget>
   <widget class="Canvas" name="Canvas_4" native="true">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>440</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sumLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>670</y>
      <width>691</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Sum</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="averageTextEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>680</y>
      <width>421</width>
      <height>131</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1192</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
